### ISSUE TITLE
Correction, variation?

### ISSUE THREAD

**Issue by nivaca** [lombardpress/lombardpress-schema#59]
Two witnesses have:
![maz](https://cloud.githubusercontent.com/assets/2309005/18488744/643a21d2-79c0-11e6-83cb-f204e4058cc9.png)

The other two have:
![sorb](https://cloud.githubusercontent.com/assets/2309005/18488759/7042c0b0-79c0-11e6-9c2c-023d75f18879.png)

I don't want to chose the "quod" as my lemma, but I don't know how to note both that:

1. the "quod" is absent from two MSS. This would be a "variation-addition" in the `rdg@type`; and
2. the "quod" is also a "correction-addition" in the other two MSS.

This is my guess, but point (1) is missing here:

```
vel accidentia in eo, et 
<app>
  <lem/>
  <rdg wit="#S #V" type="correction-addition">
    <add place="aboveLine">quod</add>
  </rdg>
</app> 
ita non Deus.

```

Any ideas?

**Comment by stenskjaer** [comment]
How about

``` xml
vel accidentia in eo, et 
<app>
  <lem wit="#S #V" type="correction-addition">
    <add place="aboveLine">quod</add>
  </lem>
  <rdg wit="#P #L" type="variation-absent" />
</app> 
ita non Deus.
```

Or:
``` xml
vel accidentia in eo, et 
<app>
  <lem>quod</lem>
  <rdg wit="#S #V" type="correction-addition">
    <add place="aboveLine">quod</add>
  </rdg>
  <rdg wit="#P #L" type="variation-absent" />
</app> 
ita non Deus.
```

This should result in 
> quod ] *add. s.l.* SV ; *om.* PL

Or just (negatively)
> quod ] *add. s.l.* SV

A side note: Here we see the problem with the negative apparatus by the way. It is ambiguous whether P and L actually have "quod" in the text (just not added), or don't have it at all...

On the mark up:
I think both should be possible, and as I remember it, that has also been the attitude among us. But the thing is, I don't really see that reflected in the schema description. No examples show approach 1, and only rarely are there examples of a text with more than one witness, so I wonder whether we should reflect these things more clearly in the guidelines?
Any thoughts, Jeff?

**Comment by nivaca** [comment]
I'm afraid this won't work. As I noted, "quod" should appear only in the 
apparatus, not in the text.

**Comment by jeffreycwitt** [comment]
But I don't think Nick wants "quod" including in the actual text, but in @stenskjaer example, "quod" appears in the lemma and would actually appear in the critical text.

So I would do:

```xml
vel accidentia in eo, et 
<app>
  <lem wit="#M #T" n="et"/>
  <rdg wit="#S #V" type="correction-addition">
    <add place="aboveLine">quod</add>
  </rdg>
</app> 
ita non Deus.
```

The `#M` and `#T` on the lemma are not necessary but they make it explicit that in M and T nothing is there. 

I'm not sure its quite right to say M and T are an instance of variation-absent, because the "absence" is relative to the critical text chosen. 

If you wanted to include "quod" in the critical text itself, then I think you should adopt Michael's second suggestion.

```xml
vel accidentia in eo, et 
<app>
  <lem>quod</lem>
  <rdg wit="#S #V" type="correction-addition">
    <add place="aboveLine">quod</add>
  </rdg>
  <rdg wit="#M #T" type="variation-absent" />
</app> 
ita non Deus.
```

We should use these examples in clarifying the guidelines. Let's not close this issue until that's been done. I will add this ticket to the 1.0.0 milestone tag.

**Comment by nivaca** [comment]
OK, that's a nice solution. Still, I was expecting somewhere the 
"variation-absent", as relative to the Edition, #S and #V don't have it.

**Comment by jeffreycwitt** [comment]
But what is the "edition"? Isn't your critical transcription your best guess as to what constitutes the "edition". 

In any case, correction-addition for S and V is enough to the tell the reader two things. First "quod" was first missing, and second then "quod" was added later. So it basically covers the idea of an "original variation-absent" within its meaning.

**Comment by nivaca** [comment]
Yep. The edition is our better understanding of all witnesses. In it, 
"quod" should be absent.
